book/chapters/chapter3/evaluation_and_benchmarking.qmd

```diff
@@ -285,6 +285,31 @@ print(plt2)
 ```
 
 
+### Confidence Intervals {#sec-resampling-ci}
+
+Confidence intervals (CIs) provide a range of values within which we can be confident that it covers the true generalization error.
```

Comment:
I would delete the first sentence. The next sentence basically says the same, but this one isn't quite accurate.